docs/modules/ROOT/pages/usage.adoc

```diff
@@ -1,19 +1,80 @@
 = Usage
 
-You need to tell RuboCop to load the RSpec extension. There are three
-ways to do this:
+You need to tell RuboCop to load the RSpec extension.
+There are three ways to do this:
 
 == RuboCop configuration file
 
-Put this into your `.rubocop.yml`.
+Put this into your `.rubocop.yml`:
 
 ----
 require: rubocop-rspec
 ----
 
+or, if you are using several extensions:
+
+----
+require:
+ - rubocop-rspec
+ - rubocop-performance
+----
+
 Now you can run `rubocop` and it will automatically load the RuboCop RSpec
 cops together with the standard cops.
 
+=== RSpec DSL configuration
+
+In case you https://github.com/rspec/rspec-core/blob/b0d0843a285693c64cdbe0c85726db155b46047e/lib/rspec/core/configuration.rb#L1122[define aliases for RSpec DSL], i.e. examples, example groups, hooks, or include example statements, you need to properly configure it so those elements are properly detected by RuboCop-RSpec.
```

Comment:
```suggestion
In case you https://github.com/rspec/rspec-core/blob/b0d0843a285693c64cdbe0c85726db155b46047e/lib/rspec/core/configuration.rb#L1122[define aliases for RSpec DSL], i.e. examples, example groups, hooks, or include example statements, you need to configure it so those elements are properly detected by RuboCop RSpec.
```
You had `properly` twice in the same sentence there.